app/src/main/java/com/iic/shopingo/ui/request_flow/Activities/RequestStateActivity.java

```diff
@@ -1,4 +1,4 @@
-package com.iic.shopingo.ui.request_flow;
+package com.iic.shopingo.ui.request_flow.Activities;
```

Comment:
Don't make packages that start with uppercase, that cause issues between different OS.